client/src/components/ui/forms/permissions/LocalPermissions.svelte

```diff
@@ -28,32 +25,32 @@
         });
 
         // No point in handling no-changes.
-        if (user.permission === permsVal) return;
+        if (payload.permission === permsVal) return;
 
         try {
             // Rebuild pseudo-user object
             await Staff.setPermission({
-                office,
-                user_id: user.id,
+                office: payload.office,
+                user_id: payload.id,
                 permission: permsVal,
             });
 
-            // Might be editing own session, might be not, reload anyway
-            await userSession.reload?.();
+            // Reload the staffList store
+            await staffList.reload?.();
         } catch (err) {
             // TODO: No permission handler
             alert(err);
         }
     }
 </script>
-<p>You are modifying {user.email}'s permissions in Office ID {office}.</p>
+<p>You are modifying {payload.email}'s permissions in Office ID {payload.office}.</p>
```

Comment:
```suggestion
<p>You are modifying {payload.email}'s permissions in {officeName}.</p>
```
I recommend replacing the office ID (which is payload.office) with the actual Office name (string) to make it more user friendly,
`$: officeName = $allOffices[office] ?? 'No office name.';`